app/build.gradle

```diff
@@ -87,4 +96,20 @@ dependencies {
     testCompile 'org.json:json:20170516'
     testCompile 'org.apache.httpcomponents:httpclient:4.5.3'
     testCompile 'org.mockito:mockito-core:2.8.9'
+
+
+    //wall of coins
+    compile 'com.squareup.retrofit2:retrofit:2.0.2'
+    compile 'com.squareup.retrofit2:converter-gson:2.0.2'
+    compile 'com.squareup.okhttp3:logging-interceptor:3.3.1'
+    compile 'com.squareup.okhttp:logging-interceptor:2.7.5'
+    compile 'com.google.code.findbugs:jsr305:2.0.3'
+    compile 'org.dashj:dashj-core:0.14.4.3'
```

Comment:
Why are you using dashj and not the pivxj that is already in the project? This will cause a lot of redundant dependencies

Reply:
We have removed dependency of 'org.dashj:dashj-core:0.14.4.3' now.

https://github.com/PIVX-Project/PIVX-Android/pull/8/commits/359d68e08713226fd43ee5d46ade5cf55b767ca7